packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -175,6 +175,26 @@ export default class FlexPluginsStart extends FlexPlugin {
     }
   }
 
+  /**
+   * Throws an error if user inputted a taken port
+   * Returns the port if available
+   *
+   * @param port
+   * @returns
+   */
+  async checkUserPortIsAvailable(): Promise<number> {
+    const flexPort = this._flags.port
```

Comment:
`const port = await findPortAvailablePort('--port', this._flags.port || '3000')`